parkings/api/operator/permit.py

```diff
@@ -1,8 +1,64 @@
-from rest_framework import mixins
+from django.db import transaction
+from django.utils.translation import ugettext_lazy as _
+from rest_framework import mixins, serializers
+from rest_framework.decorators import action
+from rest_framework.response import Response
+
+from parkings.models import Permit, PermitSeries
 
 from ..common_permit import PermitSeriesViewSet
 from .permissions import IsOperator
 
 
 class OperatorPermitSeriesViewSet(mixins.DestroyModelMixin, PermitSeriesViewSet):
     permission_classes = [IsOperator]
+
+    @action(detail=True, methods=['post'])
+    def activate(self, request, pk=None):
+        with transaction.atomic():
+            obj_to_activate = self.get_object()
+            old_actives = self.queryset.filter(active=True, owner=self.request.user)
```

Comment:
I think `self.get_queryset()` should be used here and the owner filtering should happen there.